Nomic Bot Error

Version: newopen

Error: Cannot find module '../command-processors/open-processor.js'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/nomic-bot/dist/endpoint-processors/comment-processor.js:34:22)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)

The bot has been reverted to the previous version: extensible

--- a/nomic-bot/src/endpoint-processors/comment-processor.js
+++ b/nomic-bot/src/endpoint-processors/comment-processor.js
@@ -5,7 +5,6 @@ import multi from 'multiple-methods';
 import * as logger from '../utils/logger.js';
 import * as rollProcessor from '../command-processors/roll-processor.js';
 import * as voteProcessor from '../command-processors/vote-processor.js';
-import * as openProcessor from '../command-processors/open-processor.js';
 import * as closeProcessor from '../command-processors/close-processor.js';
 import * as buyProcessor from '../command-processors/buy-processor.js';
 import * as resolveProcessor from '../command-processors/resolve-processor.js';

